.travis.yml

```diff
@@ -2,7 +2,23 @@ language: java
 # OpenTripPlanner requires Java 8 and Travis doesn't (yet) support OpenJDK 8
 jdk:
   - oraclejdk8
-  
+  - openjdk8
+  - oraclejdk9
+  - openjdk9
+# deprecated - oraclejdk10
+  - openjdk10
+  - oraclejdk11
+  - openjdk11
+  - openjdk-ea
+matrix:
+  allow_failures:
```

Comment:
Please add a comment explaining what allow_failures does. According to the help text on Travis CI: we still see pass/fail status for these sub-builds, but if they fail it does not cause the entire build to be marked as a failure.

Reply:
exactly. Allow_failures allows to test again the early access of the JDK while not making the build RED. It is good to have in order to detect early incompatibilities